tests/jenkins/run_test_amzn_linux_gpu.sh

```diff
@@ -65,5 +65,5 @@ nosetests3 --verbose tests/python/train
 #julia -e 'try Pkg.clone("MXNet"); catch end; Pkg.checkout("MXNet"); Pkg.build("MXNet"); Pkg.test("MXNet")' || exit -1
 
 echo "BUILD scala_test"
-make scalapkg
-make scalatest
+cd scala-package
+mvn integration-test -Dskip.test=false
```

Comment:
You should use -DskipTests=false. Also, why is this one "make scalatest -> integration tests" while run_test_ubuntu.sh is "make scalatest -> unit tests"?

Reply:
Since this is tested on GPU, we should enable the integration test there